templates/devshells/c/flake.nix

```diff
@@ -5,6 +5,11 @@
   inputs.flake-utils.url = "github:numtide/flake-utils";
   inputs.nickel-nix.url = "github:nickel-lang/nickel-nix";
 
+  nixConfig = {
+    extra-substituters = ["https://tweag-nickel.cachix.org"];
+    extra-trusted-public-keys = ["tweag-nickel.cachix.org-1:GIthuiK4LRgnW64ALYEoioVUQBWs0jexyoYVeLDBwRA="];
+  };
+
```

Comment:
This comes from the template directory generating the templates directories, you should commit the original file too

Reply:
nevermind, I thought we still had a `rust-devshell` directory where development is happening, and the `rust` example that was created from it :sweat_smile: